bb_wrapper/models/pagamentos.py

```diff
@@ -325,43 +388,71 @@ class LiberarPagamentos(BaseModel):
+    314: "Fim do empréstimo ocorreu durante período de suspensão ou concessão",
+    315: "Empréstimo suspenso pelo banco",
+    316: "Não averbação de contrato",
+    317: "Lote Não Aceito - Totais do Lote com Diferença",
     318: "Título Não Encontrado",
-    319: "Identificador Registro Opcional Inválido. Verifique dado informado.",
-    320: "Código Padrão Inválido. Verifique dado informado.",
-    321: "Código de Ocorrência Inválido. Verifique dado informado.",
-    322: "Complemento de Ocorrência Inválido. Verifique dado informado.",
+    319: "Identificador Registro Opcional Inválido",
+    320: "Código Padrão Inválido",
+    321: "Código de Ocorrência Inválido",
+    322: "Complemento de Ocorrência Inválido",
     323: "Alegação já Informada",
-    324: "Agência / Conta do Favorecido Substituída. Verifique dado informado.",
-    325: "Divergência entre o primeiro e último nome do beneficiário VS primeiro e último nome na Receita Federal. Verificar com beneficiário necessidade de ajustes.",  # noqa: E501
+    324: "Agência / Conta do Favorecido Substituída",
+    325: "Divergência entre o primeiro e último nome do beneficiário na Receita Federal",  # noqa
     326: "Confirmação de Antecipação de Valor",
     327: "Antecipação parcial de valor",
     328: "Boleto bloqueado na base. Não passível de pagamento.",
-    329: "Sistema em contingência – Boleto valor maior que referência. Consulte o beneficiário ou tente efetuar o pagamento mais tarde.",  # noqa: E501
-    330: "Sistema em contingência – Boleto vencido. Consulte o beneficiário ou tente efetuar o pagamento mais tarde.",  # noqa: E501
-    331: "Sistema em contingência – Boleto indexado. Consulte o beneficiário ou tente efetuar o pagamento mais tarde.",  # noqa: E501
-    332: "Beneficiário divergente. Verifique dado informado.",
-    333: "Limite de pagamentos parciais do boleto excedido. Consulte o Beneficiário do boleto.",  # noqa: E501
+    329: "Sistema em contingência – Boleto valor maior que referência",
+    330: "Sistema em contingência – Boleto vencido",
+    331: "Sistema em contingência – Boleto indexado",
+    332: "Beneficiário divergente.",
+    333: "Limite de pagamentos parciais do boleto excedido.",
     334: "Boleto já liquidado. Não passível de pagamento.",
-    999: "Consultar o Banco para detalhar o erro.",
+    999: "Consultar o Banco para detalhar o erro",
+}
+
+ERROS_ESTORNO_PAGAMENTO_DICT = {
+    1: "Conta destinatária do crédito encerrada",
+    2: "Agência ou conta destinatária do credito inválida",
+    3: "Ausência ou divergência na indicação do CPF/CNPJ",
+    4: "Msg inválida p/ o tipo de transação ou finalidade",
+    5: "Divergência na titularidade",
+    6: "Transferência insuficiente para finalidade indicada",
+    7: "Diferença a maior",
+    8: "Código identificador de transferência inválido",
+    9: "Devolução por fraude",
+    15: "Identificação deposito Judicial Inválida",
+    16: "Mensagem STRO020/PAG0116 fora do horário def do negócio",
```

Comment:
"def" seria deferimento?
```suggestion
    16: "Mensagem STRO020/PAG0116 fora do horário def do negócio",
```

Reply:
definido

Reply:
Feito